### PR TITLE
Produce Android x86_64 binaries

### PR DESCRIPTION
**Pull request by eoger**
Fixes https://github.com/mozilla/application-services/issues/426.
Let's hope the switch to OpenSSL 1.1 with its new build system done in https://github.com/mozilla/application-services/pull/474 ironed out the compilation problems described in the linked issue.

### CODE REVIEW

File: libs/README.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ This directory builds `openssl` for iOS, Android and desktop platforms.
 
 ### Supported Arch
 
-* Android: `TARGET_ARCHS=("x86" "arm64" "arm")`
+* Android: `TARGET_ARCHS=("x86" "x86_64 "arm64" "arm")`
```
Comment (vladikoff) [inline libs/README.md]: missed the `"`

File: libs/build-openssl-android.sh [hunk 2]
```diff
@@ -31,7 +31,10 @@ export PATH="$TOOLCHAIN_PATH/bin:$PATH"
 OPENSSL_OUTPUT_PATH="/tmp/openssl-$TOOLCHAIN"_$$
 mkdir -p "$OPENSSL_OUTPUT_PATH"
 
-if [ "$TOOLCHAIN" == "i686-linux-android" ]
+if [ "$TOOLCHAIN" == "x86_64-linux-android" ]
+then
+  CONFIGURE_ARCH="android64-x86_64"
```
Comment (ncalexan) [inline libs/build-openssl-android.sh]: Wow, that's an exciting name.

File: automation/taskcluster/decision_task.py [hunk 3]
```diff
@@ -193,7 +193,7 @@ def linux_build_task(name):
         })
         .with_index_and_artifacts_expire_in(build_artifacts_expire_in)
         .with_artifacts("/build/sccache.log")
-        .with_max_run_time_minutes(60)
+        .with_max_run_time_minutes(120)
```
Comment (ncalexan) [inline automation/taskcluster/decision_task.py]: Really?  Is `x86_64` particularly slow?

Comment (eoger) [reply automation/taskcluster/decision_task.py]: I could have been unlucky, but yeah the job timed out on me :/

**Comment by eoger** [comment]
Thanks Nick!

**Pull request merged**